config/production.js

```diff
@@ -18,5 +18,8 @@ module.exports = {
   },
   l10n: {
     use_local: true
+  },
+  stripe: {
+    enthusiast_plan_id: 'plan_Fc2wCyqj2Azpbm' // TODO DECIDE IF THIS SHOULD BE .ENV
```

Comment:
In production, we should be reading the `STRIPE_PROD_PUBLIC_KEY` and `STRIPE_PROD_SECRET_KEY` variables, right? Or does that happen elsewhere?

Reply:
I may be wrong, but I think we just named it `TEST` because it was initially a test, but I don't see a need to keep both sets of secrets, we can just specify one `STRIPE_SECRET_KEY` for whatever instance is running. I'm gonna rename this to make it more clear.